.eslintrc.js

```diff
@@ -0,0 +1,15 @@
+module.exports = {
```

Comment:
You do not use the same eslint rules as https://github.com/42BV/redux-mad-authentication/pull/8/files#diff-e4403a877d80de653400d88d85e4801a

Reply:
This project does not use React, therefore I've removed those rules.